### PR TITLE
Use `copy-mode -q` to leave copy mode before running a command

### LINKED ISSUE #230: "Not in a mode" error when in scroll mode
I get the "not in a mode" error when using `VimuxRunCommand` while the Vimux pane is in scroll mode.

Repro Steps:
- In neovim, run `:VimuxOpenRunner`
- in the new VImux pane run `ls ~` to get some output
- in the new Vimux pane, enter copy mode (`C-b PgUp` in default keybindings)
- In neovim, run `:VimuxRunCommand("ls")`

The following error is displayed in neovim:
```
Error detected while processing function VimuxRunCommand[10]..VimuxSendKeys[2]..VimuxTmux[8]..function VimuxRunCommand[10]..VimuxSendKeys[2]..VimuxTmux:
line    8:
E605: Exception not caught: Tmux command failed with message:not in a mode
```

I get the same error when calling the tmux cli directly:
```
> tmux send-keys -t 2 q C-u
not in a mode   
```

This seems to be related to `q C-u` being sent together. If I run them as two separate commands, there is no error:

```
> tmux send-keys -t 2 q
> tmux send-keys -t 2 C-u
```

I realize splitting into separate commands might not be compatible with `VimuxResetSequence` being configured as a single string.

Here are some versions. 
```
> tmux -V
tmux 3.4

> nvim -v
NVIM v0.10.1
Build type: RelWithDebInfo
LuaJIT 2.1.1707061634
Run "nvim -V1 -v" for more info

> alacritty --version
alacritty 0.13.2

vimux: (I don't know how to get neovim to tell me this, but it should be the latest master)
```

I'm happy to answer any other questions or make a screen recording if needed.

### PR DESCRIPTION
**Pull request by mvanderkamp**
Fixes #230 

Four years ago `tmux` introduced a `-q` flag to `copy-mode`: https://github.com/tmux/tmux/commit/06c3079d66929e0c71575e877098fc533ae5f4a5

Using `copy-mode -q` on the runner instead of relying on the `q` mapping, which could be remapped, should be more reliable.

This updates the default value of `VimuxResetSequence` from `"q C-u"` to simply `"C-u"`.

This will pose problems for users on a `tmux` version older than the introduction of the `-q` flag (before version 3.2a). We could perhaps catch the unknown flag error and try using `q` in those cases.

### CODE REVIEW

**Review by alerque — APPROVED** [review]

**Comment by mvanderkamp** [comment]
I thought it would be prudent to put in a fallback for older versions so the runner prompt is still cleared. I haven't found the precise cause of this issue in tmux. Since we haven't changed our logic it seems likely that this stems from a recent tmux release and that the `q C-u` reset sequence used to work properly.

**Comment by mvanderkamp** [comment]
@alerque Looks like I need a fresh approval, sorry about that!

**Review by alerque — APPROVED** [review]

**Comment by alerque** [comment]
No worries, tweaking things in PRs to get them as right as we know how before merging is the right thing to do. That being said I've watched your contributions for a while and don't think there is any need for the friction of waiting for a rubber stamp from me. I just upgraded your access to this repo to *Admin* level so you should be able to skip the 1-review-needed requirement. Have fun.

Everybody else: please welcome your new overlord!

**Pull request merged**
